.craft.yml

```diff
@@ -0,0 +1,4 @@
+changelogPolicy: auto
+preReleaseCommand: pwsh scripts/bump-version.ps1
+targets:
+  - name: github
```

Comment:
Since we're not using release registry here, this PR is unblocked.

On a follow up PR, we can add a new target, like this:
https://github.com/getsentry/sentry-unreal/blob/4027d198c6ab87016d95665fcf9af1df3cac4a31/.craft.yml#L12-L13

But basically as:

```
  - name: registry
    sdks:
      github:getsentry/sentry-godot:
```


That is, once this is merged: https://github.com/getsentry/sentry-release-registry/pull/174